### PR TITLE
Adds utilities for downloading and working with JRA55 data

### LINKED ISSUE #53: Nuke `neverworld_simulation.jl`
@sandreza @simone-silvestri 

Unfortunately nobody started using `neverworld_simulation` in `ClimaOcean.jl`. Since it takes work to maintain, I'm going to nuke it. Raising this issue in case you have changed your mind and would like to conduct your neverworld work here after all.

Here's the code: https://github.com/CliMA/ClimaOcean.jl/blob/main/src/IdealizedSimulations/neverworld_simulation.jl

### PR DESCRIPTION
**Pull request by glwagner**
Also closes #53 .

### CODE REVIEW

File: src/DataWrangling/JRA55.jl [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+    isfile(filename) || download(url, filename)
+
+    ds = Dataset(filename)
+    data = ds[variable_name][:, :, time_indices]
+
+    # Make source field
+    λ = ds["lon_bnds"][1, :]
+    φ = ds["lat_bnds"][1, :]
+    times = ds["time"][time_indices]
```
Comment (simone-silvestri) [inline src/DataWrangling/JRA55.jl:44]: what are the contents of this ds file? Should we write it somewhere?

**Review by navidcy — COMMENTED** [review]
I think that somewhere around where RYF urls are defined let's add a comment that this is the Repeat-Year-Forcing from JRE55-do dataset and cite paper by Kial et al., doi:[10.1016/j.ocemod.2019.101557](https://doi.org/10.1016/j.ocemod.2019.101557).

File: src/DataWrangling/JRA55.jl [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    jra55_field_time_series(name, architecture=CPU();
+                            time_indices = :,    
+                            url = urls[name],
+                            filename = filenames[name],
+                            variable_name = variable_names[name])
+
+Return a FieldTimeSeries representing JRA55 data at the interface between the
+atmosphere and ocean.
+"""
+function jra55_field_time_series(name, arch=CPU();
```
Comment (simone-silvestri) [inline src/DataWrangling/JRA55.jl:30]: We need to explain what `name` is

File: src/DataWrangling/JRA55.jl (hunk 1 above)
Comment (glwagner) [reply src/DataWrangling/JRA55.jl]: That's not a bad idea, maybe a comment here...

Comment (glwagner) [reply src/DataWrangling/JRA55.jl]: done

File: src/DataWrangling/JRA55.jl (hunk 2 above)
Comment (glwagner) [reply src/DataWrangling/JRA55.jl]: done

**Review by navidcy — APPROVED** [review]

File: src/IdealizedSimulations/neverworld_simulation.jl
Comment (navidcy) [inline src/IdealizedSimulations/neverworld_simulation.jl:1]: @glwagner I want to resurrect these and use them
I'll put in a different repo?

**Pull request merged**